webpack/translations-loader.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * {
+ *   name: 'test app',
+ *   author: {
+ *     title: 'the author',
+ *     value: 'mr programmer'
+ *   },
+ *   app: {
+ *     instructions: 'install',
+ *     steps: {
+ *       click: 'this button'
+ *     }
+ *   }
+ * }
+ *
+ * becomes
+ *
+ * {
+ *   name: 'test app',
+ *   author: 'mr programmer',
+ *   app.instructions: 'install',
+ *   app.steps.click: 'this button'
+ * }
+ */
+function translationFlatten (object, currentKeys = []) {
+  const res = {}
+
+  Object.keys(object).map(
+    key => {
```

Comment:
I know JS allows no brackets, but I don't like that for readability.

`set.map(() => {`
`set.map(arg => {` vs `set.map((arg) => {`
`set.map((arg1, arg2) => {`

Suddenly no brackets?! 😞 

I feel this is going to be fixed in standard/prettier one day 😆